src/lib/logger.js

```diff
@@ -42,39 +42,63 @@ function getDate(): string {
   return (new Date()).toISOString();
 }
 
-class Logger {
-  static container: Container;
 
-  static log(...params: Array<any>) {
+export class Logger {
+
+   log(...params: Array<any>) {
     if (settings.SHOW_VERBOSE_DEVICE_LOGS) {
-      Logger._log(`[${getDate()}]`, _transform(...params));
+      this._log(`[${getDate()}]`, _transform(...params));
     }
   }
 
-  static info(...params: Array<any>) {
-    Logger._log(
+   info(...params: Array<any>) {
+    this._log(
       `[${getDate()}]`,
       chalk.cyan(_transform(...params)),
     );
   }
 
-  static warn(...params: Array<any>) {
-    Logger._log(
+   warn(...params: Array<any>) {
+    this._log(
       `[${getDate()}]`,
       chalk.yellow(_transform(...params)),
     );
   }
 
-  static error(...params: Array<any>) {
-    Logger._log(
+   error(...params: Array<any>) {
+    this._log(
       `[${getDate()}]`,
       chalk.red(_transform(...params)),
     );
   }
 
-  static _log(...params: Array<any>): Function {
-    return Logger.container.constitute('LOGGING_FUNCTION')(...params);
+  setContainer (container) {
```

Comment:
I don't think we need this. We should just use `console.log` for the default logger and get rid of `LOGGING_FUNCTION` since you can override the whole class if you want.